### PR TITLE
add login widget based on parties.json

### PR DESCRIPTION
**Pull request by alexmatson-da**
Also modifies `OnboardingTile` to support an array of `Tile` UI components, each tile being separated by a small gap. See screenshot for example

Screen Shot 2021-01-07 at 3 59 14 PM

### CODE REVIEW

**Review by alexgraham-da — APPROVED** [review]
this is awesome! I tested it out, works great. will definitely make testing/demoing a lot easier

File: ui/src/components/LoginScreen.css [hunk 1]
```diff
@@ -2,3 +2,40 @@
...
+    margin: 0;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    align-self: flex-end;
+}
+
+.parties-login .custom-file-upload .icon {
+    margin-right: var(--spacing-s);
```
Comment (Sarahb-da) [inline ui/src/components/LoginScreen.css:40]: 👍

File: ui/src/components/common/OnboardingTile.css [hunk 2]
```diff
@@ -12,11 +19,29 @@
 }
 
 .onboarding-tile .onboarding-tile-content .icon {
-    width: var(--spacing-m);
-    height: var(--spacing-m);
+    /* width: var(--spacing-m);
+    height: var(--spacing-m); */
     margin-right: var(--spacing-s);
```
Comment (Sarahb-da) [inline ui/src/components/common/OnboardingTile.css:24]: Possibly forgot to remove comment?

File: ui/src/components/common/OnboardingTile.css [hunk 3]
```diff
@@ -12,11 +19,29 @@
...
+
+.onboarding-tile .onboarding-tile-content .field .ui {
+    border: none;
+    background-color: var(--grey1);
+}
+
+.onboarding-tile .onboarding-tile-content .ui.form input,
+.onboarding-tile .onboarding-tile-content .ui.form .field > .selection.dropdown {
+    width: 100%;
+    background: #fff;
```
Comment (Sarahb-da) [inline ui/src/components/common/OnboardingTile.css:39]: FWIW we have a `--white` now

**Review by Sarahb-da — APPROVED** [review]
Looks good, this is nice work!

**Pull request merged**